eng/pipeline.yml

```diff
@@ -36,128 +30,104 @@ jobs:
-        # Will eventually change this to two BYOC pools.
-        # agent pool can't be read from a user-defined variable (Azure DevOps limitation)
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
           name: NetCore-Public
           demands: ImageOverride -equals windows.vs2022preview.amd64.Open
         ${{ if eq(variables['System.TeamProject'], 'internal') }}:
           name: NetCore1ESPool-Internal
           demands: ImageOverride -equals windows.vs2022preview.amd64
       variables:
-        - name: Codeql.Enabled
-          value: true
-        # needed for signing
-        - name: _TeamName
-          value: DotNetCore
+      - name: Codeql.Enabled
+        value: true
+      - name: _TeamName
+        value: DotNetCore
+      - name: _SignType
+        value: real
+      - name: _SignArgs
+        value: ''
+      - name: _PublishArgs
+        value: ''
+      - name: _OfficialBuildIdArgs
+        value: ''
+      - name: _Platform
+        value: x86
+      - name: _PlatformArgs
+        value: /p:Platform=$(_Platform)
+      - name: _PublicBuildPipeline
+        value: true
+      - name: _TestHelixAgentPool
+        value: 'Windows.10.Amd64.ClientRS5.Open'
+      - name: _HelixStagingDir
+        value: $(BUILD.STAGINGDIRECTORY)\helix\functests
+      - name: _HelixSource
+        value: ${{ parameters.repoName }}/$(Build.SourceBranch)
+      - name: _HelixToken
+        value: ''
+      - name: _HelixCreator
+        value: ${{ parameters.repoName }}
+      - name: _programfilesx86
+        value: ${Env:ProgramFiles(x86)}/dotnet
+      - name: _programfiles
+        value: ${Env:ProgramFiles}/dotnet
+      - ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+        - name: _InternalRuntimeDownloadArgs
+          value: ''
+      - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+        - group: DotNetBuilds storage account read tokens
+        - group: AzureDevOps-Artifact-Feeds-Pats
+        - name: _InternalRuntimeDownloadArgs
+          value: >-
+            /p:DotNetRuntimeSourceFeed=https://dotnetbuilds.blob.core.windows.net/internal /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
+      - ${{ if eq(parameters.runAsPublic, 'false') }}:
         - name: _SignType
           value: real
+        - group: DotNet-HelixApi-Access
         - name: _SignArgs
-          value: ''
+          value: /p:DotNetSignType=$(_SignType) /p:TeamName=$(_TeamName)
         - name: _PublishArgs
-          value: ''
+          value: /p:DotNetPublishUsingPipelines=true
         - name: _OfficialBuildIdArgs
-          value: ''
-        - name: _Platform
-          value: x86
-        - name: _PlatformArgs
-          value: /p:Platform=$(_Platform)
-        - name: _PublicBuildPipeline  # We will run Helix tests when building in the open, but do not repeat when building and publishing again using the internal build-pipeline
```

Comment:
The migration tool unfortunately remove comments. Consider re-adding these comments if they are helpful for understanding what is going on

Reply:
Sure, I have re-added some helpful comments